### PR TITLE
Reversed  action

### PR DESCRIPTION
**Pull request by osamaashraf6**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Add the functionality for the reversed action
- [x] Handle on which the button will appear depends on the active key of the rocket👌
- [x] Fetch the data of rockets from the API
- [x] Display the data at the rocket page using useSelector 💯.
- [x] Add the configureStore for the redux.

## Description of the task to be completed 👎 

- [x] Add the functionality for the reversed action💯.
- [x] Handle on which the button will appear depends on the active key of the rocket⤵️ 👎
- [x] Fetch the data of rockets from the API👍.
- [x] Fetch the data of rockets from the API

  
## How should it be tested

Clone the repository
```
git clone https://github.com/osamaashraf6/Space-X-React-Group
```
Navigate to the folder
```
cd Space-X-React-Group
```
```
Checkout the branch
```
git checkout reversed__action
```

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-- [x] Add the rockets and missions at separate components
-- [x] Install the react-router-dom and use the version@6.👌
+- [x] Add the functionality for the reversed rocket.
+- [x] Identify which button will appear depend on the active of the rocket👌
```
Comment (momotsow) [inline README.md:82]: Great job updating your ReadMe

**Review by momotsow — APPROVED** [review]
Hi @osamaashraf6,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 
- ReadMe updated ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: README.md (hunk 1 above)
Comment (osamaashraf6) [reply README.md:82]: - Thanks  @momotsow 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊

**Comment by osamaashraf6** [comment]
- Thanks @momotsow 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊

**Pull request merged**